src/ontology/mondo-edit.obo

```diff
@@ -566171,6 +566185,14 @@ is_a: MONDO:0005027 {source="PMID:28276060"} ! epilepsy
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/8471" xsd:anyURI
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/8992" xsd:anyURI
 
+[Term]
+id: MONDO:0100581
+name: ocular growth disorder
+def: "Aberrant development of the eye resulting in significant shortening or elongation, a disorder that affects the final ocular dimensions" [https://clinicalgenome.org/affiliation/40077/]
```

Comment:
this definition does not refer to a disease. Could you please update to "an eye disorder which is characterized by ..."